### PR TITLE
Change Duzen to Siezen in password dialogs

### LINKED ISSUE #435: Sie-Form auf den Passwort-Setzen-Seiten
https://match4healthcare.slack.com/archives/C010K4T5YGZ/p1587019922015800

Auf der “Passwort setzen” Seite der Registrierungen, sollten wir lieber Sietzen anstatt weiter zu dutzen und “Liebe Vertrer*in einer Einrichtung,” oder sowas schreiben
Bisheriger Text
Liebe*r Helfer*in,
vielen Dank für deine Registrierung. Auf dieser Seite kannst du ein Passwort für deinen Account erstellen und dich damit einloggen.

### PR DESCRIPTION
**Pull request by Baschdl**
Fixes #435

### CODE REVIEW

File: backend/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -528,18 +528,18 @@ msgstr "Please correct the errors shown below"
 msgid "Passwort gesetzt"
 msgstr "Password reset"
 
-msgid " Ihr Passwort wurde gesetzt. Sie können sich jetzt einloggen "
+msgid " Ihr Passwort wurde gesetzt. Sie können sich jetzt einloggen."
 msgstr " Your password was changed. You can login now "
```
Comment (bjrne) [inline backend/locale/en/LC_MESSAGES/django.po]: Also add the dot here?
```suggestion
msgstr " Your password was changed. You can login now. "
```

**Review by bjrne — APPROVED** [review]
Seems fine :)

File: backend/locale/en/LC_MESSAGES/django.po (hunk 1 above)
Comment (bjrne) [reply backend/locale/en/LC_MESSAGES/django.po]: Are the spaces intentional? If so, the original suggenstion also replaces the trailing space with a dot... maybe a layout issue?

Comment (Baschdl) [reply backend/locale/en/LC_MESSAGES/django.po]: Intentional in the way that they are like this in the html

**Pull request merged**